app/models/user.rb

```diff
@@ -1,3 +1,2 @@
 class User < ApplicationRecord
-  has_many :tickets, dependent: :nullify, inverse_of: :user
```

Comment:
Try to add https://www.pluralsight.com/guides/ruby-on-rails-nested-attributes instead of removing this reference.
Nested attributes is not a good practice, usually we use form object, but let it be.